frontend/lib/strapi-sdk/operations/getProductList.graphql

```diff
@@ -1,5 +1,9 @@
 query getProductList($filters: ProductListFiltersInput) {
-   productLists(pagination: { limit: 1 }, filters: $filters) {
+   productLists(
+      pagination: { limit: 1 }
+      filters: $filters
+      publicationState: LIVE
```

Comment:
I'm not seeing any reference to this filter / prop on the backend. Will this just work? Or is there some part of the Strapi API I'm missing?

I'm mostly concerned that this will return 0 results if no product-list have this property set.

Reply:
@evannoronha I'm not sure what you're referring to, what do you mean exactly?